### PR TITLE
feat(banks-users): added API functions for GET Accs and GET Trans

### PR DESCRIPTION
**Pull request by Jules-NC**
Added BanksUser API functions calls

## Description
Two new functions are added in BanksUser.ts:
- `getAccountsByID` corresponding to: https://developers.algoan.com/api/#operation/getbankAccounts
- `getTransactionsByIds` corresponding to: https://developers.algoan.com/api/#operation/getTransactions

## Motivation and Context
Not all functions related to the aggregation manager API are present. They should be added. In addition, they have become essential for us.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

**Comment by Jules-NC** [comment]
@ccoeurderoy

File: src/core/BanksUser.ts [hunk 1]
```diff
@@ -86,6 +86,45 @@ export class BanksUser {
...
 
+  /**
+   * Fetch all Accounts related to a BankUserID
+   *
+   * @param id Id of the BanksUser to fetch
+   * @param requestBuilder Service account request builder
+   *
+   * @returns Array of BanksUserAccount
+   */
+  public static async getAccountsByID(id: string, requestBuilder: RequestBuilder): Promise<BanksUserAccount[]> {
```
Comment (ccoeurderoy) [inline src/core/BanksUser.ts:97]: If you add the `ById` suffix, it means that you are looking for a specific account

```suggestion
  public static async getAccounts(id: string, requestBuilder: RequestBuilder): Promise<BanksUserAccount[]> {
```

File: src/core/BanksUser.ts [hunk 2]
```diff
@@ -86,6 +86,45 @@ export class BanksUser {
...
+  /**
+   * Fetch all Transactions related to a BankUserID and an AccountID
+   *
+   * @param bankUserId Id of the BanksUser to fetch
+   * @param accountId Id of the Account to fetch
+   * @param requestBuilder Service account request builder
+   *
+   * @returns Array of BanksUserTransaction
+   */
+  public static async getTransactionsByIds(
```
Comment (ccoeurderoy) [inline src/core/BanksUser.ts:115]: Same here

```suggestion
  public static async getTransactionsPerAccounts(
```

**Comment by Jules-NC** [comment]
Done.
@ccoeurderoy

**Pull request merged**